controllers/generic.go

```diff
@@ -0,0 +1,64 @@
+package controllers
```

Comment:
we should have a better name for this file, I believe `healthcheck.go` would be just fine

Reply:
Doing that!